### ISSUE TITLE
Login currently not working

### ISSUE THREAD

**Issue by torbenberger** [InstaPy/InstaPy#2965]
I get `Timed out with failure while explicitly waiting until visibility of element located!` followed by `Wrong login data!` when i try to login a new account, anyone else getting this issue?

(Login data is correct of course)

**Comment by uluQulu** [comment]
@torbenberger 
>Timed out with failure while explicitly waiting until visibility of element located!

That message says that `Get App` offer did not pop up in there.

>Wrong login data!

Well, it is an issue.
Can you run a graphical session and provide feedback on what you see happens?

------

Cheers 😁
------

**Comment by neppo75** [comment]
i got the same issue

**Comment by uluQulu** [comment]
@neppo75 
https://github.com/timgrossmann/InstaPy/issues/2965#issuecomment-423129434

**Comment by torbenberger** [comment]
> @torbenberger
> 
> > Timed out with failure while explicitly waiting until visibility of element located!
> 
> That message says that `Get App` offer did not pop up in there.
> 
> > Wrong login data!
> 
> Well, it is an issue.
> Can you run a graphical session and provide feedback on what you see happens?
> 
> ## Cheers 😁

already did that yesterday, it starts the browser, shows IG Homepage, shows Google Homepage, shows IG Homepage again, and crashes

**Comment by uluQulu** [comment]
@torbenberger 
If it crashes show the console output;
If it shows `Wrong login`, provide more feedback.

I cannot see your screen so you must write what happens.
+ It should, open homepage- ig,com
+ then click on the `Login in` button [at bottom of page] which should navigate to `ig.com/accounts/login`
+ now only username and password boxes should appear [no ANY Facebook or other button]
 and a Log in button again
+ after filling username & password, should click on Log in
+ then `Get App` offer might pop up or not AND it will be dismissed as pops up

Which of the steps it does and which ones it does not?

----

Cheers 😁
-------

**Comment by torbenberger** [comment]
Okay now it crashes directly after opening ig.com homepage with no JS errors in Chrome but this error in python console:

```
OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
INFO [2018-09-21 13:47:12] [username]  Session started!
oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo

INFO [2018-09-21 13:47:43] [username]  Sessional Live Report:
        |> LIKED 0 images  |  ALREADY LIKED: 0
        |> COMMENTED on 0 images
        |> FOLLOWED 0 users  |  ALREADY FOLLOWED: 0
        |> UNFOLLOWED 0 users
        |> INAPPROPRIATE images: 0
        |> NOT VALID users: 0
currently FOLLOWING 0 users & has got 0 FOLLOWERS


OOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOOO
INFO [2018-09-21 13:47:43] [username]  Session ended!
oooooooooooooooooooooooooooooooooooooooooooooooooooooooooooo



Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1321, in getresponse
    response.begin()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 296, in begin
    version, status, reason = self._read_status()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 265, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
http.client.RemoteDisconnected: Remote end closed connection without response

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "quickstart.py", line 110, in <module>
    bot()
  File "quickstart.py", line 53, in bot
    session.login()
  File "/Users/torbenberger/Projekte/torbenberger/instapy/InstaPy/instapy/instapy.py", line 382, in login
    self.bypass_suspicious_attempt):
  File "/Users/torbenberger/Projekte/torbenberger/instapy/InstaPy/instapy/login_util.py", line 107, in login_user
    web_address_navigator(browser, ig_homepage)
  File "/Users/torbenberger/Projekte/torbenberger/instapy/InstaPy/instapy/util.py", line 658, in web_address_navigator
    browser.get(link)
  File "/usr/local/lib/python3.7/site-packages/selenium-3.14.0-py3.7.egg/selenium/webdriver/remote/webdriver.py", line 332, in get
    self.execute(Command.GET, {'url': url})
  File "/usr/local/lib/python3.7/site-packages/selenium-3.14.0-py3.7.egg/selenium/webdriver/remote/webdriver.py", line 318, in execute
    response = self.command_executor.execute(driver_command, params)
  File "/usr/local/lib/python3.7/site-packages/selenium-3.14.0-py3.7.egg/selenium/webdriver/remote/remote_connection.py", line 375, in execute
    return self._request(command_info[0], url, body=data)
  File "/usr/local/lib/python3.7/site-packages/selenium-3.14.0-py3.7.egg/selenium/webdriver/remote/remote_connection.py", line 397, in _request
    resp = self._conn.request(method, url, body=body, headers=headers)
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/request.py", line 72, in request
    **urlopen_kw)
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/request.py", line 150, in request_encode_body
    return self.urlopen(method, url, **extra_kw)
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/poolmanager.py", line 322, in urlopen
    response = conn.urlopen(method, u.request_uri, **kw)
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/connectionpool.py", line 638, in urlopen
    _stacktrace=sys.exc_info()[2])
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/util/retry.py", line 367, in increment
    raise six.reraise(type(error), error, _stacktrace)
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/packages/six.py", line 685, in reraise
    raise value.with_traceback(tb)
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/connectionpool.py", line 600, in urlopen
    chunked=chunked)
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/connectionpool.py", line 384, in _make_request
    six.raise_from(e, None)
  File "<string>", line 2, in raise_from
  File "/usr/local/lib/python3.7/site-packages/urllib3-1.23-py3.7.egg/urllib3/connectionpool.py", line 380, in _make_request
    httplib_response = conn.getresponse()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 1321, in getresponse
    response.begin()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 296, in begin
    version, status, reason = self._read_status()
  File "/usr/local/Cellar/python/3.7.0/Frameworks/Python.framework/Versions/3.7/lib/python3.7/http/client.py", line 265, in _read_status
    raise RemoteDisconnected("Remote end closed connection without"
urllib3.exceptions.ProtocolError: ('Connection aborted.', RemoteDisconnected('Remote end closed connection without response'))
```

also it does not login nor create a cookie file in the logs folder

**Comment by alexdelorenzo** [comment]
This issue was "solved" in #3029 by logging in with a Windows computer and copying the cookie.

#3029 was closed, on the assumption that their IP was banned. 

However, I ran into the same issue and "solved" it while using the same IP address to connect to Instagram: I ran the script on a Mac and copied the cookie file over. I am not IP banned.

What can be discerned:
- First login with Instapy can fail on Linux machines.
- First login with Instapy might not fail on Windows or Mac machines.
- Perhaps this is triggered by Instagram itself.
- The error is not necessarily the result of an IP ban.

**Comment by Olcaey** [comment]
Hey, I am having the same issue and even though I tried with different IPs and removing the cookie files, it still generates the same error @torbenberger wrote down. Could there be any other reason for this error?
